### PR TITLE
Alternative starting point?

### PR DESCRIPTION
**Pull request by ArceusMaxis**
felt like this would make it more easy for new peeps

### CODE REVIEW

**Comment by gfodor** [comment]
I think what I'd prefer here is to just let people go to webspaces.space - since that is where I plan on adding all of the onboarding documentation for people who aren't interested in working on the webspace engine. The README here should primarily be about working on the engine - but we can def add a line directing people to webspaces.space. I'll merge this PR if you make that change, thank you!

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This is the engine for [Webspaces](https://webspaces.space), which allows you to create 3D worlds using just HTML.
 
+## Notice
+
+If you want to get started with **webspaces.space** instead of the engine it works on, please go to [webpsace-sdk](https://github.com/webspace-sdk/webspace-sdk.github.io)
```
Comment (gfodor) [inline README.md]: this doesn't look right - the link is pointing to the repo for webspaces.space - I was suggesting we do something like:

`If you aren't interested working on the Webspace Engine, but wish to create your own webspace, please go to [webspaces.space]`

And dropping the other changes.

Comment (ArceusMaxis) [reply README.md]: Ho, okay, just a minute

File: README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 This is the engine for [Webspaces](https://webspaces.space), which allows you to create 3D worlds using just HTML.
 
+## Notice
```
Comment (gfodor) [inline README.md]: sorry to be pedantic, but we can probably drop this header

Comment (ArceusMaxis) [reply README.md]: Sure! its alright, i need to learn the job properly if i wanna help you right? XD

**Review by gfodor — APPROVED** [review]

**Pull request merged**